src/components/containers/TraceRequiredPanel.js

```diff
@@ -12,17 +15,14 @@ class TraceRequiredPanel extends Component {
   }
   checkTraceExistence() {
     const {visible} = this.props;
-    const {layout} = this.context;
+    const {fullData} = this.context;
     const {hasTraces} = this.state;
 
     if (visible) {
       /**
        * Check if there is any trace data
        */
-      if (
-        Object.keys(layout.xaxis).length === 0 ||
-        Object.keys(layout.yaxis).length === 0
-      ) {
+      if (fullData.filter(trace => trace.visible).length === 0) {
         if (hasTraces) {
```

Comment:
I would have put `fullData.filter(trace => trace.visible).length === 0 && hasTraces`, but it's ok :)